src/store/utils.js

```diff
@@ -78,6 +78,7 @@ export function initEditorSections(data, sectionsNames){
                 }),
                 is_dataset: false
             },
+            additionalInformation: JSON.parse(JSON.stringify(data.metadata)),
```

Comment:
Not the most elegant but fine :). I would only have taken the fields I need using destructuring instead of copying the full metadata object. A bit more optimized but not a major thing.